host/setup.py

```diff
@@ -23,12 +23,13 @@ def read(fname):
             'greatfet_spiflash = greatfet.commands.greatfet_spiflash:main',
             'greatfet_DS18B20 = greatfet.commands.greatfet_DS18B20:main',
             'greatfet_i2c = greatfet.commands.greatfet_i2c:main',
+            'greatfet_shell = greatfet.commands.greatfet_shell:main',
         ],
     },
     #author='', #TODO: Figure out whose name should go here!
     #author_email='',
     tests_require=[''],
-    install_requires=['pyusb'],
+    install_requires=['pyusb', 'IPython', 'pygreat'],
```

Comment:
Consider using `extras_require` for `IPython`.  I don't think installing GreatFET should force the user to install IPython since it's only a convenience feature used by `greatfet_shell`.  You can catch `ImportError` in `greatfet_shell` and tell the user to install it, or provide a different behavior.